pilot/pkg/networking/core/v1alpha3/listener.go

```diff
@@ -1869,17 +1869,11 @@ func buildHTTPConnectionManager(pluginParams *plugin.InputParams, httpOpts *http
 		}
 
 		acc := &accesslog.AccessLog{
-			Name: wellknown.FileAccessLog,
+			Name:       wellknown.FileAccessLog,
+			ConfigType: &accesslog.AccessLog_TypedConfig{TypedConfig: util.MessageToAny(fl)},
```

Comment:
My concern is will this break the compatibility when users just upgrade the control plane？

Reply:
Envoy supported typed config for years now. Why do you think it's causing compatibility issues?

Reply:
Because i remember we  have a flag protect this, if users upgerade with old proxy  still there, would this bring trouble for them?

Reply:
They would need a very old proxy (before 1.0). We don't support these ancient proxies anyways.

Reply:
IC, thanks for clarify

Reply:
The flag was only added because of a performance regression which was fixed long ago by the way, not for old proxy compatibility